README.md

```diff
@@ -66,6 +66,7 @@ On the host via the script
       -i <TEXT>  [optional] Specify image ID used within Sysdig (ex: -i '<64 hex characters>')
       -d <PATH>  [optional] Specify image digest (ex: -d 'sha256:<64 hex characters>')
       -m <PATH>  [optional] Path to Docker image manifest (ex: -m ./manifest.json)
+      -l  [optional] To invoke curl commands as secure (locked) connections
```

Comment:
'secure' and 'locked' are not synonyms. `-l` is fine as the argument, but I think it isn't useful to mention "(locked)" in the description.